### PR TITLE
[JS->TS] Migrate get_auction_elements.js

### PR DESCRIPTION
**Pull request by bh2smith**
Very easy switch over. Noticed that colours weren't appearing as I would have expected in my terminal. Partial fulfillment of #330

**Test Plan** Take a while, but

```
npx truffle exec build/common/scripts/get_auction_elements.js --network rinkeby
```

Will print a bunch of orders and then say something like `Found 1218 orders`. One example order looks like

```
{
  user: 0x9bc7569755aab9ddb9be1aaf7b28a90565d79c06
  sellTokenBalance: 0
  buyToken: 7
  sellToken: 1
  validFrom: Mon May 25 2020 12:00:00 GMT+0200 (Central European Summer Time)
  validUntil: Thu Aug 17 42800 23:10:00 GMT+0200 (Central European Summer Time)
  price: Sell 13 * 10^35 for at least 34 * 10^37
  remaining: 13 * 10^35
}
```

### CODE REVIEW

File: scripts/get_auction_elements.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  ) {
+    return COLORS.YELLOW;
+  } else {
+    return COLORS.NONE;
+  }
+};
+
+const printOrder = function (order: Order<BN>, currentBatchId: number) {
+  /* eslint-disable no-console */
+  console.log("{");
```
Comment (fleupold) [inline scripts/get_auction_elements.ts:79]: should this take a logger now that we have it?

File: scripts/get_auction_elements.ts [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        argv.tokens.split(",").map((t: string) => parseInt(t.trim())),
+      );
+      auctionElementsDecoded = auctionElementsDecoded.filter(
+        (order) => tokens.has(order.buyToken) && tokens.has(order.sellToken),
+      );
+    }
+
+    auctionElementsDecoded.forEach((order) => printOrder(order, batchId));
+    // eslint-disable-next-line no-console
+    console.log(`Found ${auctionElementsDecoded.length} orders`);
```
Comment (fleupold) [inline scripts/get_auction_elements.ts]: ditto

**Review by fleupold — APPROVED** [review]
Like the easy tag. This script is arguably not that useful anymore now that fetching the orderbook via onchain state takes forever, but given you already converted it it's nice to kepp. Maybe we will reuse it one day...

File: scripts/get_auction_elements.ts (hunk 1 above)
Comment (bh2smith) [reply scripts/get_auction_elements.ts:79]: It doesn’t look good with all of the time stamps and info. Just seems unnecessarily wide with the logger.  Also can’t be copied easily... not that we would ever use it.

File: scripts/get_auction_elements.ts (hunk 2 above)
Comment (bh2smith) [reply scripts/get_auction_elements.ts]: This I would say should be a log.

**Pull request merged**